src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/directionissued/DirectionIssuedMidEventHandlerTest.java

```diff
@@ -211,14 +212,25 @@ private void verifyTemplateBody(String image, String expectedName, String templa
     }
 
     @Test
-    public void shouldErrorWhenDirectionTypeIsProvideInformationAndNoDueDate() {
+    public void shouldErrorWhenDirectionTypeIsGrantExtensionAndNoDueDate() {
         sscsCaseData.setDirectionDueDate(null);
-        sscsCaseData.getDirectionTypeDl().setValue(new DynamicListItem(DirectionType.PROVIDE_INFORMATION.toString(), "appeal To Proceed"));
+        sscsCaseData.getDirectionTypeDl().setValue(new DynamicListItem(DirectionType.GRANT_EXTENSION.toString(), "Allow time extension"));
         PreSubmitCallbackResponse<SscsCaseData> response = handler.handle(MID_EVENT, callback, USER_AUTHORISATION);
         assertEquals(1, response.getErrors().size());
 
         assertEquals("Please populate the direction due date", response.getErrors().toArray()[0]);
     }
 
+
```

Comment:
Some useless space line can remove because it may cause you fail the format check, I have failed before.